docs/operation/lerobot_guide.rst

```diff
@@ -0,0 +1,306 @@
+==========================
+LeRobot X Aloha User Guide
+==========================
+
+Virtual Environment Setup
+=========================
+
+Containerization is crucial for running machine learning models to avoid dependency conflicts. You can either use a Virtual Environment (venv) or Conda for this purpose.
```

Comment:
```suggestion
Containerization is crucial for running machine learning models to avoid dependency conflicts.
You can either use a Virtual Environment (venv) or Conda for this purpose.
```